lib/dpul_collections_web/live/item_live.ex

```diff
@@ -593,13 +616,15 @@ defmodule DpulCollectionsWeb.ItemLive do
   def thumbs(assigns) do
     ~H"""
     <div class="pr-2 pb-2">
-      <img
-        class="h-full w-full object-cover"
-        src={"#{@thumb}/full/350,465/0/default.jpg"}
-        alt={"image #{@thumb_num}"}
-        style="
-          background-color: lightgray;"
-      />
+      <a href={"#{@viewer_url}/#{@thumb_num + 1}"} }>
+        <img
+          class="h-full w-full object-cover"
+          src={"#{@thumb}/full/350,465/0/default.jpg"}
+          alt={"image #{@thumb_num}"}
+          style="
+            background-color: lightgray;"
+        />
+      </a>
```

Comment:
```suggestion
      <.link patch={"#{@viewer_url}/#{@thumb_num + 1}"} }>
        <img
          class="h-full w-full object-cover"
          src={"#{@thumb}/full/350,465/0/default.jpg"}
          alt={"image #{@thumb_num}"}
          style="
            background-color: lightgray;"
        />
      </.link>
```

Using a `.link` here should prevent the reload that happens.